docs/languages/cpp.md

```diff
@@ -33,21 +31,19 @@ If you run into any issues or have suggestions for the Microsoft C/C++ extension
 
 With the C/C++ extension installed, open a folder that contains your C/C++ source code. VS Code will place various settings files into a `.vscode` subfolder.
 
-**Note**: The C/C++ extension does not include a C++ compiler or debugger. You will need to install these tools or use those already installed on your computer. Popular C++ compilers are [MinGW](http://www.mingw.org/) for Windows, [XCode](https://developer.apple.com/xcode/) for macOS, and [GCC](https://gcc.gnu.org/) on Linux. Also make sure your compiler executable is on your platform path for VS Code to find.
+**Note**: The C/C++ extension does not include a C++ compiler or debugger. You will need to install these tools or use those already installed on your computer. Popular C++ compilers are [mingw-w64](http://www.mingw-w64.org/) for Windows, Clang for [XCode](https://developer.apple.com/xcode/) for macOS, and [GCC](https://gcc.gnu.org/) on Linux. Also make sure your compiler executable is on your platform path for VS Code to find.
 
-### IntelliSense
+### Configuring IntelliSense
 
-**To enable code completion and navigation, you will need to generate a `c_cpp_properties.json` file:**
+The extension will attempt to determine your folder's basic configuration info based on compilers it finds on your system. If for any reason, that configuration is incomplete, you can generate a `c_cpp_properties.json` file by running the **C/Cpp: Edit configurations...** command from the **Command Palette** (`kb(workbench.action.showCommands)` and add the missing information.
```

Comment:
...it finds on your system, including macOS, Linux, Windows (MSVC), Windows subsystem for Linux (WSL), MinGW. and Cygwin.

Reply:
I don't think this additional verbosity adds to the topic.

Reply:
I'd like to call out the platforms supported, otherwise users would have never thought of using VS Code for WSL, for example. And it makes clear which ones we don't support :)